models/classes/Lists/Business/Service/ClassMetadataService.php

```diff
@@ -184,12 +181,16 @@ private function getPropertyValues(core_kernel_classes_Property $property): ?arr
 
     private function isTextWidget(core_kernel_classes_Property $property): bool
     {
-        return in_array($property->getWidget()->getUri(), self::TEXT_WIDGETS);
+        return ($property->getWidget()->getUri())
+            ? in_array($property->getWidget()->getUri(), self::TEXT_WIDGETS, true)
+            : false;
```

Comment:
```suggestion
       $widgetUri = $property->getWidget()->getUri();
       
        return $widgetUri
            ? in_array($widgetUri, self::TEXT_WIDGETS, true)
            : false;
```
To avoid repeat twice same method.

Reply:
fixed